crates/services/relayer/src/service.rs

```diff
@@ -348,14 +351,21 @@ pub fn new_service<D>(database: D, config: Config) -> anyhow::Result<Service<D>>
 where
     D: RelayerDb + Clone + 'static,
 {
-    let url = config.relayer.clone().ok_or_else(|| {
-        anyhow::anyhow!(
-            "Tried to start Relayer without setting an eth_client in the config"
-        )
-    })?;
-    // TODO: Does this handle https?
-    let http = Http::new(url);
-    let eth_node = Provider::new(http);
+    let urls = config
+        .relayer
+        .clone()
+        .ok_or_else(|| {
+            anyhow::anyhow!(
+                "Tried to start Relayer without setting an eth_client in the config"
+            )
+        })?
+        .into_iter()
+        .map(|url| {
+            // TODO: Does this handle https?
```

Comment:
Has this been resolved yet? Otherwise it sounds like we should create a follow-up issue to look into this.

Reply:
@xgreenx are we already using https somewhere and we can validate this or do you want me to test it ?

Reply:
yes we are already using https in prod

Reply:
Nice, then I assume we can remove the comment.

Reply:
Done :)